app/scripts/components/ui/biospecimen/biospecimen.controllers.ts

```diff
@@ -56,11 +56,8 @@ module ngApp.components.ui.biospecimen.controllers {
     }
 
     displayBioSpecimenDocumentRow(key, value): boolean {
-      if (key.toLowerCase() === "expanded") {
-        return false;
-      }
-
-      if (key.toLowerCase() === "submitter_id") {
+      const hiddenKeys = ['expanded', 'submitter_id', 'state', 'updated_datetime', 'created_datetime'];
```

Comment:
do you need hidden keys? can you just remove them from the query and the template?